src/persistent-storage/wallets.ts

```diff
@@ -260,6 +262,70 @@ export const persistAddressesMetadata = async (walletId: string, addressesMetada
   await persistWalletsMetadata(walletsMetadata)
 }
 
+export const persistContact = async (contactData: ContactFormData) => {
```

Comment:
Nit: why not adding those function to a dedicated `contacts.ts` file?

Reply:
Good question. I chose to leave them in this file because I made `contacts` be part of the wallet metadata object. Since all wallet metadata functions are in this file, I decided to keep these here as well. Not a strong opinion on this.